javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,45 @@
+const width = 12;
+const height = 14;
+const area = width * height;
+
+console.log('Value of the area:', area);
+console.log('Type of the area:', typeof area);
+
+const bill = 199;
+const payment = 50;
+const change = bill - payment;
+
+console.log('Value of the change:', change);
+console.log('Type of change:', typeof change);
+
+const quizzes = 65;
+const midterm = 87;
+const final = 72;
+const grade = (quizzes + midterm + final) / 3;
+
+console.log('Value of the grade:', grade);
+console.log('Type of grade:', typeof grade);
+
+const firstName = 'Tiffany';
+const lastName = 'Brown';
+const fullName = firstName + ' ' + lastName;
+
+console.log('Value of fullName:', fullName);
+console.log('Type of fullName:', typeof fullName);
+
+const ph = 5;
+const isAcidic = ph < 7;
+
+console.log('Value of the isAcidic:', isAcidic);
+console.log('Type of isAcidic:', typeof isAcidic);
+
+const headCount = 10;
+const isSparta = headCount === 300;
+
+console.log('Value of isSparta:', isSparta);
+console.log('Type of isSparta:', typeof isSparta);
+
+const motto = fullName + ' ' + ' is the GOAT';
```

Comment:
This should be broken up into 2 steps. You should originally initialize `motto` and assign it a value of `fullName`.

Then, using the `+=` operator, you would concatenate the string back to `motto` and reassign